packages/libp2p/test/connection-manager/resolver.spec.ts

```diff
@@ -205,50 +160,16 @@ describe('dialing (resolvable addresses)', () => {
     await deferred.promise
   })
 
-  it('resolves a dnsaddr recursively not failing if one address fails to resolve', async () => {
```

Comment:
can you explain why we need to remove this test? it feels like a use-case not quite covered by `ma.resolve`

Reply:
Because recursively resolving DNSADDRs is done by `@multiformats/multiaddr` and not this module.

Reply:
lol. i guess that's clear enough.